CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ CHANGELOG
 
 **CHANGES**
 
+- Upgrade EFA installer to version 1.10.1 to support CentOS 8:
```

Comment:
the no change is with respect to 1.10.0 or with respect to what was deployed with pcluster 2.9.1? we should compare with the latter. Also I'd list the added support for centos 8 as a subitem